### ISSUE TITLE
Add skeval.model_selection module with cross_val_score and train_test_split

### ISSUE THREAD

**Issue by direkkakkar319-ops** [skeval-ai/skeval#20]
## Summary
There is no standard way to measure generalization. Add a `model_selection` module wrapping sklearn's cross-validation utilities.

## Changes required
Create `src/skeval/model_selection.py` with:

**`cross_val_score(clf, X, y, cv=5)`**
- Wraps `sklearn.model_selection.StratifiedKFold`
- Returns a numpy array of per-fold accuracy scores

**`train_test_split(X, y, test_size=0.2, stratify=True, random_state=None)`**
- Thin wrapper over sklearn's `train_test_split` with `stratify=y` by default

## Target API
```python
from skeval.model_selection import cross_val_score
scores = cross_val_score(clf, sentences, labels, cv=5)
print(f"{scores.mean():.2f} ± {scores.std():.2f}")
```

## Acceptance criteria
- `cross_val_score` works end-to-end with `SentenceClassifier`
- Exported from `skeval.model_selection`

**Milestone:** v0.2.0
